src/test/java/seedu/placebook/logic/UiStubFactory.java

```diff
@@ -0,0 +1,43 @@
+package seedu.placebook.logic;
+
+import javafx.stage.Stage;
+import seedu.placebook.ui.Ui;
+
+public class UiStubFactory {
+    private static class positiveConfirmationUi implements Ui {
+
+        @Override
+        public void start(Stage primaryStage) {
+
+        }
+
+        @Override
+        public boolean showDeleteDialogAndWait(String contentText) {
+            return true;
+        }
+    }
+
+    private static class negativeConfirmationUi implements Ui {
+
+        @Override
+        public void start(Stage primaryStage) {
+
+        }
+
+        @Override
+        public boolean showDeleteDialogAndWait(String contentText) {
+            return false;
+        }
+    }
+
+    private static final Ui POSITIVE_CONFIRMATION_UI = new positiveConfirmationUi();
```

Comment:
Since these are static already, I am not sure if the above classes need to be static, please clarify if I am wrong

Reply:
I think both will work? but I kinda dont want to give the impression that I can have different Factories so I chose to keep everything static here.

Reply:
Makes sense